requirements/licenses.ini

```diff
@@ -9,6 +9,7 @@ authorized_licenses:
     mozilla public license 2.0 (mpl 2.0)
     PSF
     Python Software Foundation
+    ISC License (ISCL)
```

Comment:
@rcoup this looks ok to me but you asked to be in the loop on license changes. 👍 / 👎  ?

Reply:
Thanks. Yes, all good 👍